pyfunnel/pyfunnel.py

```diff
@@ -89,6 +89,8 @@
         data[s] = dict(x=[], y=[])
         with open(vars(args)[s]) as csvfile:
             spamreader = csv.reader(csvfile)
+            if (len(next(spamreader)) != 2):
```

Comment:
I believe that if you use `next` here, the next iteration on the iterator (for loop below) starts at index 1 and not 0, i.e., the first row is skipped.
Why not test `len(spamreader[0])`? Or each line within the for loop: `len(row)`?